backend/user/middleware.go

```diff
@@ -215,13 +215,24 @@ func NewUserMiddleware(queries *sqlc.Queries, remoteCache *remotecache.Client, l
 					return i.OrgID
 				})
 			} else {
-				info, err := auth0Client.GetUserInfoForAuthHeader(ctx, ctx.GetHeader("Authorization"))
+				info, err := auth0Client.GetUser(ctx, ctx.GetString(auth0.CtxUserID))
 				if err != nil {
 					return nil, err
 				}
-				u.PersonID = info.Sub
+				u.PersonID = info.UserId
```

Comment:
Long term we probably shouldn't use the auth0 ID as the primary key. But that can work for now.

Reply:
Once we get another key to use, no problem implementing it. We aren't actually using this for much except that.